<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Читать файл:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditRead"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="dialogReadButton">
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="readButton">
     <property name="text">
      <string>Читать</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Записать файл:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditWrite"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="dialogButton">
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="generateButton">
     <property name="text">
      <string>Записать</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Первый чекбокс</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>Второй чекбокс</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>Третий чек бокс</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>398</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEditCB1"/>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEditCB2"/>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEditCB3"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
